docs/sources/features/datasources/azuremonitor.md

```diff
@@ -263,11 +263,13 @@ You can also select additional number value columns (with, or without multiple d
 ```kusto
 Perf
 | where $__timeFilter(TimeGenerated)
-| summarize Samples=count(), AvgValue=avg(CounterValue)
+| summarize Samples=count(), ["Avg Value"]=avg(CounterValue)
     by bin(TimeGenerated, $__interval), Computer, CounterName, InstanceName
 | order by TimeGenerated asc
 ```
 
+> **Tip**: In the above query, the Kusto syntax `Samples=count()` and `["Avg Value"]=...` is used to rename those columns — the second syntax allowing for the space. This will change the name of the metric that Grafana uses, and as a result, things like series legends and table columns will match what you specify. In this case, `Samples` will be displayed instead of `_count`.
```

Comment:
```suggestion
> **Tip**: In the above query, the Kusto syntax `Samples=count()` and `["Avg Value"]=...` is used to rename those columns — the second syntax allowing for the space. This changes the name of the metric that Grafana uses, and as a result, things like series legends and table columns will match what you specify. Here `Samples` is displayed instead of `_count`.
```